apps/helptheducks.com/src/app/page.tsx

```diff
@@ -36,7 +34,7 @@ const Hero = () => {
     <section
       className="md:min-h-[90vh] w-full text-white lg:bg-center md:bg-[40%] bg-[45%] bg-cover flex flex-col"
       style={{
-        backgroundImage: `url(${heroDuckImg})`,
+        backgroundImage: `url(${heroDuckImg.src})`,
```

Comment:
hmmm didn't realize this would work is the image wasn't in the public dir, interesting